index.html

```diff
@@ -0,0 +1,30 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <title>Getting Started in Front End Dev in 4 Web Pages</title>
+    <link rel="stylesheet" href="./styles/main.css">
+</head>
+
+<body>
+    <!-- Front page overview: intro graph and flexbox overview of html, css & command-line -->
+    <main id="container">
+        <!-- Page title and summary -->
+        <header class="title">Placeholder</header>
+
```

Comment:
Please add a navbar header :)

Reply:
Good idea! I'm thinking of adding semantic tag <nav>.